packaging/common/substitute.pl

```diff
@@ -125,92 +266,75 @@ sub fix_pkg_rev {
         $replacement_strings{"%%PKG_REV%%"} =
             fix_pkg_rev($replacement_strings{"%%PKG_REV%%"}, "deb");
```

Comment:
let's not determine the distro here... because it matters little except to fill the variable DISTRO.